src/main/java/org/eln2/mc/common/blocks/BlockRegistry.kt

```diff
@@ -55,4 +52,6 @@ object BlockRegistry {
     val WIRE_CELL = registerCellBlock("wire", eln2Tab) { WireCellBlock() }
     val VOLTAGE_SOURCE_CELL = registerCellBlock("voltage_source", eln2Tab) { VoltageSourceCellBlock() }
     val GROUND_CELL = registerCellBlock("ground", eln2Tab) { GroundCellBlock() }
+    val CAPACITOR_CELL = registerCellBlock("capacitor", eln2Tab) { CapacitorCellBlock() }
+    val INDUCTOR_CELL = registerCellBlock("inductor", eln2Tab) { InductorCellBlock() }
```

Comment:
This should be it's own pull request. Please don't add fixes, a naming refactor, plus plotter changes in one big request.